lib/phare/check/jshint.rb

```diff
@@ -25,7 +25,13 @@ def command
     protected
 
       def excluded_files
-        [] # TODO: Fetch the exclude list from .jshintignore
+        return [] unless configuration_file
+
+        configuration_file.split("\n").flat_map { |path| Dir.glob(path) }
```

Comment:
I think we should use a regex here to cover all possible `newline` characters. What do you think?